pkg/cmd/experimental/syncgroups/syncgroups.go

```diff
@@ -138,8 +139,8 @@ func NewCmdSyncGroups(name, fullName string, f *clientcmd.Factory, out io.Writer
 
 	cmd.Flags().StringVar(&whitelistFile, "whitelist", whitelistFile, "path to the group whitelist")
 	cmd.Flags().StringVar(&configFile, "sync-config", configFile, "path to the sync config")
-	cmd.Flags().StringVar(&typeArg, "type", typeArg, "type of group used to locate LDAP group UIDs: "+strings.Join(AllowedSourceTypes, ","))
-	cmd.Flags().BoolVar(&options.Confirm, "confirm", false, "if true, modify OpenShift groups; if false, display groups")
+	cmd.Flags().StringVar(&typeArg, "type", typeArg, "which groups whitelist entries refer to: "+strings.Join(AllowedSourceTypes, ","))
```

Comment:
@deads2k @liggitt naming thoughts appreciated. I liked `source` because this is the source of the groups being synced, as in, it's the source of their names. I can see the negatives to `source` however. 

We could require people use the full name (`group/groupname`) in the whitelist and then we could programatically determine this ... and I think right now we require them *not* to add `group/`, that should be doc'ed.